components/collection/drop/GenerativeLayout.vue

```diff
@@ -102,10 +106,13 @@ import type {
 import { useCollectionMinimal } from '@/components/collection/utils/useCollectionDetails'
 import useCursorDropEvents from '@/composables/party/useCursorDropEvents'
 import { DropEventType } from '@/composables/party/types'
+import { GenerativePreviewItem } from '@/composables/drop/useGenerativePreview'
 
 const props = withDefaults(
   defineProps<{
     collectionId: string
+    amountToMint?: number
```

Comment:
eslint "requires default value to be set"

Reply:
fixed